plugins/git/functions

```diff
@@ -64,6 +64,12 @@ git_build_app_repo() {
   suppress_output git submodule update --init --recursive
   find . -name .git -prune -exec rm -rf {} \; > /dev/null
 
+  if use_git_worktree; then
```

Comment:
Doesn't this delete the worktree before it gets used below?

Reply:
After deleting `.git` directory, `git worktree prune` command removes this worktree on original(`$DOKKU_ROOT/$APP`) repository. But actual directory(`/tmp/dokku_git.XXXX` which will mounted on `/tmp/app` in docker container) is remained.
I tested on my local machine, Build was fine.

Reply:
Ah! I see what you mean now, I thought this meant directories were staying in `/tmp` :)